examples/Bunch/include/simulation_defines/param/laserConfig.param

```diff
@@ -56,11 +56,13 @@ BOOST_CONSTEXPR_OR_CONST float_64 AMPLITUDE_SI = _A0 * UNITCONV_A0_to_Amplitude_
  *  unit: seconds (1 sigma) */
 BOOST_CONSTEXPR_OR_CONST float_64 PULSE_LENGTH_SI = 10.615e-15 / 4.0; //5.0e-15; //25.0e-15 / 1.17741;
 
-/** beam waist: distance from the axis where the pulse intensity (E^2)
+/** beam waist: distance from the axis where the electric field (E)
  *              decreases to its 1/e^2-th part,
```

Comment:
according to your  first (not numbered) equation, at `x = w_0` the Electric field is at 1/e-th. The intensity is at 1/e^2-th.